.github/workflows/test_python_cplusplus.yml

```diff
@@ -149,25 +137,9 @@ jobs:
       run: |
         brew install hdf5 swig gcc cppcheck libomp
 
-    - name: Build suitesparse
-      run: |
-        scripts/buildSuiteSparse.sh
-
-    - name: Build sundials
-      run: |
-        scripts/buildSundials.sh
-
-    - name: Build cpputest
-      run: |
-        scripts/buildCpputest.sh
-
-    - name: Build BNG
-      run: |
-        scripts/buildBNGL.sh
-
     - name: Build AMICI
       run: |
-        scripts/buildAmici.sh
+        scripts/buildAll.sh
```

Comment:
I actually prefer the separate steps. Comes with some redundancies, but at least it's super easy to see which steps succeeded without scrolling through walls of text :grimacing:

Reply:
Can we somehow induce text folding from within a script?

Reply:
https://github.community/t/has-github-action-somthing-like-travis-fold/16841 apparently

Reply:
Okay split `buildAmici` and `buildDependencies`, ThirdParty stuff should be pretty stable by now, right?

Reply:
Sounds good.

Reply:
Actually, in case of building problems, the test_install is probably the first one to fail, so lets just split up that one and keep the compact version elsewhere.

Reply:
ok